runners/direct-java/src/test/java/org/apache/beam/runners/direct/MultiStepCombineTest.java

```diff
@@ -64,7 +64,7 @@
   private transient KvCoder<String, Long> combinedCoder =
       KvCoder.of(StringUtf8Coder.of(), VarLongCoder.of());
 
-  @Test
+  @Test(timeout = 10000)
```

Comment:
This is unrelated. It may shrink the code review latency to try to separate little changes and just get them merged.

Reply:
Fixed.